hack/generate_addons.sh

```diff
@@ -18,9 +18,9 @@ kustomize_build() {
   kustomize build $ADDONS_DIR/kustomize/overlays/$1 > $ADDONS_DIR/$1.yaml
 }
 
-helm_template eks aws-node-termination-handler 0.13.3
-helm_template autoscaler cluster-autoscaler 9.10
-helm_template nvdp nvidia-device-plugin 0.10.0
+helm_template eks aws-node-termination-handler 1.15.0
```

Comment:
I get:

```
Error: failed to download "eks/aws-node-termination-handler" at version "1.15.0"
```

I think the latest helm chart version is 0.18.4

Reply:
Fixed to 0.18.4